docs/FAQ.rst

```diff
@@ -1023,17 +1023,17 @@ to your ~/GNUstep/Defaults/Windowmaker file.
 How do I get the classic NeXT(tm)-like style back?
 ..................................................
 
-Open WPrefs and go under the ``Other Configurations`` tab. Under ``Title Bar
-Style``, select the classic look.
+Open WPrefs and navigate to the ``Other Configurations`` tab with the scroll bar. Within the ``Titlebar
```

Comment:
Lines should not exceed 80 characters, just to be coherent with rest of the file.

Reply:
Thanks, I'll make the change.